README.md

```diff
@@ -1,2 +1,28 @@
-# opentelemetry
-Get started with our Elastic Distros of OpenTelemetry
+# Elastic OpenTelemetry Distributions
+
+[OpenTelemetry](https://opentelemetry.io/docs/) is a vendor- and tool-neutral observability framework for collecting, processing, and exporting telemetry data.
+You can send your telemetry data to Elastic Observability using OpenTelemetry the following ways:
+
+- Use the Elastic Distribution for the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) to collect and send logs and host metrics to [Elastic Cloud](https://cloud.elastic.co/).
+- Use Elastic Distributions for OpenTelemetry Java, .NET, Node.js, and Python to instrument your applications and send logs, traces, and metrics to [Elastic Cloud](https://cloud.elastic.co/).
+- Configure a custom collector or contrib Distribution for the OpenTelemetry Collector to collect logs and metrics and send them to Elastic Observability.
```

Comment:
What do you think about something that follows this format: name the product, explain its relationship to the Otel ecosystem, and explain what it does? Something like this (though I'm not quite sure how to handle the last item yet):

>You can send your telemetry data to Elastic Observability using OpenTelemetry the following ways:
>
>- Use the **Elastic Distribution for OpenTelemetry Collector**, an extension of the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector), to collect and send logs and host metrics to [Elastic Cloud](https://cloud.elastic.co/).
>- Use the **Elastic Distributions for OpenTelemetry**, extensions of select [OpenTelemetry language SDKs](https://opentelemetry.io/docs/languages/), to instrument your applications and send logs, traces, and metrics to [Elastic Cloud](https://cloud.elastic.co/). The following languages are currently supported: Java, .NET, Node.js, and Python.
>- Configure your own [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or the ["contrib" distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib) to collect logs and metrics and send them to Elastic Observability.

After discussing with the APM team, we decided on describing the Elastic distros as "extensions" of the OpenTelemetry language SDKs. (From [one APM engineer](https://github.com/elastic/elastic-otel-dotnet/pull/127#discussion_r1681384136): "I think its best to say we are an open extension of the OpenTelemetry .NET SDK and are therefore not limiting anything from the standard SDK.") I'm not sure if that's the right term for the collector, too.

Reply:
>- Configure your own [custom collector](https://opentelemetry.io/docs/collector/custom-collector/) or the ["contrib" distribution](https://github.com/open-telemetry/opentelemetry-collector-contrib) to collect logs and metrics and send them to Elastic Observability.

I'm not sure if my edits to this item are appropriate. Here's what I was thinking:

* I _think_ by "custom collector" we mean something that the user has built themselves (or I guess anyone other than OpenTelemetry has built).
* The OpenTelemetry term "contrib" is confusing to me. Are we talking about an official OpenTelemetry Collector Contrib? Or is this a _type_ of custom collector? Or something else?

Reply:
I believe the [contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib) is just another distro of the otel collector that users might be using and could configure to send their data to Elastic.

Reply:
> I believe the [contrib distribution](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib) is just another distro of the otel collector that users might be using and could configure to send their data to Elastic.

https://github.com/open-telemetry/opentelemetry-collector-contrib/discussions/6971#discussioncomment-1878280

Reply:
[Core Distribution](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol/manifest.yaml) <---- [Custom Distro](https://opentelemetry.io/docs/collector/custom-collector/) ------> [Contrib Distro](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml) 
Essential components<---- Pick n' mix------> Everything

Reply:
We should refer to these in this manner:
upstream
- core distro
- custom distro
- contrib distro
- Elastic Distro 

We should emphasize where possible that the [Elastic Distro](https://github.com/elastic/elastic-agent/tree/main/internal/pkg/otel) is a supported distribution of the OpenTelemetry Collector with mostly upstream components